python/lsst/obs/cfht/megacamMapper.py

```diff
@@ -109,10 +114,12 @@ def __init__(self, **kwargs):
 
         MegacamMapper._nbit_id = 64 - (MegacamMapper._nbit_tract + 2*MegacamMapper._nbit_patch
                                        + MegacamMapper._nbit_filter)
-
-        if len(afwImageUtils.Filter.getNames()) >= 2**MegacamMapper._nbit_filter:
-            raise RuntimeError("You have more filters defined than fit into the %d bits allocated" %
-                               MegacamMapper._nbit_filter)
+        with warnings.catch_warnings():
+            # surpress Filter warnings; we already know this is deprecated
+            warnings.simplefilter('ignore', category=FutureWarning)
+            if len(afwImageUtils.Filter.getNames()) >= 2**MegacamMapper._nbit_filter:
+                raise RuntimeError("You have more filters defined than fit into the %d bits allocated" %
```

Comment:
Please don't use % based formatting

Reply:
I don't want to modify the existing code, it just got tabbed-in one level due to the added context manager.

Reply:
That's fair